lib/ws2/auth.js

```diff
@@ -19,22 +19,13 @@ const send = require('./send')
  */
 const auth = (state = {}, { dms = 0, calc = 0 } = {}) => {
   const { ev } = state
-
   const authParams = getAuthParams.get(state, dms, calc)
 
-  return new Promise((resolve, reject) => {
-    ev.once('event:auth:success', () => {
-      debug('authenticated')
-      resolve()
-    })
-
-    ev.once('event:auth:error', err => {
-      debug('error authenticating: %j', err)
-      reject(err)
-    })
+  send(state, authParams)
 
-    send(state, authParams)
-  })
+  return watchResponse(ev, 'event:auth:success', 'event:auth:error')
+    .then(() => { debug('authenticated') })
```

Comment:
this will always return auth operation as a success and will never reject on auth error